src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/globalnav/GlobalNavigationPageObject.java

```diff
@@ -124,7 +139,7 @@ public HomePageObject clickWikiaLogo() {
     if (!"prod".equals(environment) && !environment.contains("dev")) {
       WebDriverWait wait = new WebDriverWait(driver, 5);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "wait" which hides the field declared at line 72. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)